plugin/helpers_plugin_test.go

```diff
@@ -103,3 +104,30 @@ func TestInstallPluginFromURL(t *testing.T) {
 		assert.Equal(t, "received 404 status code while downloading plugin from server", err.Error())
 	})
 }
+
+func TestGetPluginAssetURL(t *testing.T) {
```

Comment:
This is a great unit test, but as noted above, won't catch the case of a plugin trying to compile to actually use this method and finding it missing from the interface. Can you replicate something like this over in `app/plugin_api_test.go`, where you'll note we actually compile plugins on demand to exercise? You might need to setup a slightly different pattern to test the helpers, but all the foundational bits are there.

Reply:
@lieut-data Okay, I will work on that and reach out if I need any help on it.

Reply:
@lieut-data I have implemented all the feedback but this part is not clear yet to me, would you be kind to help out?